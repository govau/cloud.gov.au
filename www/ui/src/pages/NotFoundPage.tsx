import * as React from "react";
import Helmet from "react-helmet";

interface Props {}

const NotFoundPage: React.StatelessComponent<Props> = () => (
  <React.Fragment>
    <Helmet>
      <title>404 page not found2</title>
    </Helmet>
    <h2>Page not found</h2>
    <p>That's all we know.</p>
  </React.Fragment>
);

export default NotFoundPage;
