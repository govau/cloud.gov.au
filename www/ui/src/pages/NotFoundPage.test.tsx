import * as React from "react";
import * as renderer from "react-test-renderer";

import NotFoundPage from "./NotFoundPage";

it("renders without crashing", () => {
  const tree = renderer.create(<NotFoundPage />).toJSON();
  expect(tree).toMatchSnapshot();
});
