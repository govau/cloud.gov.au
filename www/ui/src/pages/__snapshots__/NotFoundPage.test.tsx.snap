// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
Array [
  <h1>
    Page not found
</h1>,
  <p>
    That's all we know.
</p>,
]
`;
